### PR TITLE
User management - users now have First and Last name

### PR DESCRIPTION
**Pull request by petrakh**
Please review 😉 

Should be able to merge

### CODE REVIEW

**Review by i-hardy — APPROVED** [review]
:shipit:

**Review by memunyawiri — APPROVED** [review]
:shipit:

**Review by paulmillen — COMMENTED** [review]
:shipit:

File: spec/features/user_management_spec.rb [hunk 1]
```diff
@@ -1,9 +1,13 @@
 require 'rails_helper'
 
-def sign_up(email: 'barney@barney.com',
+def sign_up(first_name: 'Barney',
```
Comment (dbugsy) [inline spec/features/user_management_spec.rb:3]: I'd like to see more tests here - how are you handling users with only 1 name? What happens if a user only fills one field, or no fields?

**Review by dbugsy — APPROVED** [review]
I think this is fine to merge, but leave the branch open and add some more test cases. :shipit: :shipit:

**Pull request merged**